glue/viewers/profile/qt/slice_widget.py

```diff
@@ -0,0 +1,136 @@
+from glue.core.coordinates import LegacyCoordinates
+from glue.core.coordinate_helpers import dependent_axes, world_axis
+from glue.viewers.common.qt.data_slice_widget import SliceWidget
+from glue.viewers.image.state import AggregateSlice
+from glue.utils.decorators import avoid_circular
+from glue.core.data_derived import IndexedData, SlicedData
+
+__all__ = ['ProfileMultiSliceWidgetHelper']
+
+
+class ProfileMultiSliceWidgetHelper(object):
```

Comment:
Hard to see here, but what are the main differences compared to the image multi-slice widget helper class? Can they share a common base class to reduce duplication?

Reply:
I just compared the `glue/viewers/profile/qt/slice_widget.py` to `glue/viewers/image/qt/slice_widget.py` and found there are 19 differences. In essence they are not really structured exactly the same, although they look very similar. For example, the `image` one has the `y_att` which the `profile` one does not have. Also some code snippet that works for the former does not work for the latter, and needs to be tweaked to make things work.

Reply:
My reason for asking is because I want to also add sliders to the 3-d viewer :) I'll try and take a look at the differences later today (but fine to leave separate for now)

Reply:
I see

Reply:
Sure, that would be really cool